types/rest/rest.go

```diff
@@ -228,19 +228,23 @@ func ParseHTTPArgs(r *http.Request) (tags []string, page, limit int, err error)
 		if key == "page" || key == "limit" {
 			continue
 		}
+
 		var value string
 		value, err = url.QueryUnescape(values[0])
 		if err != nil {
 			return tags, page, limit, err
 		}
 
-		var tag string
-		if key == types.TxHeightKey {
-			tag = fmt.Sprintf("%s=%s", key, value)
+		// Parse request from swagger
+		// tag="action=send&sender=cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc"
+		if key == "tag" {
```

Comment:
I'm not 100% I get this, is for supporting a GET query parameter `tag` that contains itself other parameters urlencoded?
Do we have a use-case for that?

Reply:
Just for calling `/txs` through swagger as description in https://github.com/cosmos/cosmos-sdk/issues/3990#issuecomment-477882596

Reply:
We should probably fix the swagger file instead of the implementation. This is a public API, introducing too much context specific logic will make it complex

Reply:
agree with @sabau here. We should fix the swagger instead of introducing extra changes

Reply:
I'll try to follow the discussion on the issue, seems most of the context is there

Reply:
My first  solution was https://github.com/cosmos/cosmos-sdk/issues/3990#issuecomment-477563984, and @alexanderbez had different opinion https://github.com/cosmos/cosmos-sdk/issues/3990#issuecomment-477642090.

We can discuss in #3990. @sabau @fedekunze